back-end/objects/RModelWrapper.py

```diff
@@ -330,13 +314,34 @@ def init_predefined_model(self, network_type, pretrained):
                 3, 64, kernel_size=3, stride=1, padding=1, bias=False
             )
             model.maxpool = torch.nn.MaxPool2d(kernel_size=3, stride=1, padding=1)
-        elif network_type == "alexnet":
-            model = torchvision.models.alexnet(
-                pretrained=pretrained, num_classes=self.num_classes
-            )
+        elif network_type in MODEL2INIT:
+            init_func, weights = MODEL2INIT[network_type]
+            weights = weights if pretrained else None
+            model = init_func(weights=weights)
+            if self.num_classes != 1000:
```

Comment:
We defined `IMAGENET_OUTPUT_SIZE = 1000` in this file. Maybe we can use this variable instead (with another name if needed).